src/js/read.js

```diff
@@ -8,10 +8,27 @@ if (!readJSConfig.cb){ readJSConfig.cb = function(){
     Read.js assumes:
     1) The article width never exceeds the viewport width
 */
+
+function get_text(element) {
+    "use strict";
+    var ret = "";
+    var length = element.childNodes.length;
+    for(var i = 0; i < length; i++) {
+        var node = element.childNodes[i];
+        if(node.nodeType !== 8) {
+            ret += node.nodeType !== 1 ? node.nodeValue : get_text(node);
+        }
+    }
+    return ret;
+}
+
 (function(){
     "use strict";
     // Set the name of the hidden property and the change event for visibility
-    var hidden, visibilityChange; 
+    var hidden, visibilityChange;
+    var wordCount = get_text(document.getElementsByClassName("story-body")[0]).split(" ").length; //will only work if the DOM element contains text only. Consider modifying for flexibility.
```

Comment:
The DOM node to track get's passed as configuration object.
The hard coding of the element CSS selector here should be reconsidered to use the configuration object by delaying when wordCount gets defined in the handleLoad method.